dataset/__init__.py

```diff
@@ -1,40 +1,34 @@
-from dataset.huggingface_datasets import (
+from dataset.dataset_loaders import (
     CnndmDataset,
     MultinewsDataset,
     SamsumDataset,
     XsumDataset,
     PubmedqaDataset,
     MlsumDataset,
-)
-from dataset.non_huggingface_datasets import (
     ScisummnetDataset,
     SummscreenDataset,
     QMsumDataset,
     ArxivDataset,
 )
 
 
-SUPPORTED_HF_DATASETS = [
+SUPPORTED_SUMM_DATASETS = [
     CnndmDataset,
     MultinewsDataset,
     SamsumDataset,
     XsumDataset,
     PubmedqaDataset,
     MlsumDataset,
-]
-SUPPORTED_NON_HF_DATASETS = [
     ScisummnetDataset,
     SummscreenDataset,
     QMsumDataset,
     ArxivDataset,
 ]
-SUPPORTED_SUMM_DATASETS = SUPPORTED_HF_DATASETS + SUPPORTED_NON_HF_DATASETS
 
 
 def list_all_datasets():
-    all_dataset_tuples = []
+    all_datasets = []
     for ds in SUPPORTED_SUMM_DATASETS:
-        ds_obj = ds()
-        all_dataset_tuples.append((ds, ds_obj.description))
```

Comment:
Are we only showing the datasets name now?

Reply:
Previously, we would have to instantiate datasets in order to get their descriptions. This would entail downloading the datasets and loading them into memory. This will take upwards of 45 min due to large datasets such as Mlsum and Arxiv (the assumption here being, a fair amount of users calling this function haven't downloaded any of the datasets previously hence not being cached).

I wanted to use this function in the notebook, hence motivating the change.

My current thought is to have the dataset name and description as class properties defined for each dataset class. This way we do not need to rely on the initialization method to set these properties. We can then get these properties without the overhead of downloading and loading the dataset content. What do you think?

Reply:
> have the dataset name and description as class properties defined for each dataset class. This way we do not need to rely on the initialization method to set these properties. We can then get these properties without the overhead of downloading and loading the dataset content. 

Yes, this is a great idea, we are already doing it for the models I think, you can refer to those and see how it's done.

Reply:
Thanks. I'll peep it